docs/docs/guides/troubleshooting.mdx

```diff
@@ -102,21 +102,22 @@ After that make sure to run your package manager again, either `yarn` or `npm in
 
 **Solution:** Please upgrade to at least React Native 0.72 or downgrade to an older version of Reanimated 3.
 
-#### Supported Rect Native versions for Reanimated 3
-
-| Reanimated                  | New Architecture (fabric) enabled | Minimum React-native | Maximum React-native |
-| --------------------------- | --------------------------------- | -------------------- | -------------------- |
-| from **3.1.x** to **3.5.x** | ✅                                | 0.72                 | 0.72                 |
-| from **3.4.x** to **3.5.x** | ❌                                | 0.66                 | 0.72                 |
-| from **3.1.x** to **3.3.x** | ❌                                | 0.63                 | 0.72                 |
-
-#### Supported Rect Native versions for Reanimated 2
+#### Supported React Native versions - New Architecture enabled
```

Comment:
```suggestion
#### Supported React Native versions - New Architecture (Fabric) enabled
```

Reply:
I think we don't have to mention fabric two times, so I've left the second titile unchanged:

#### Supported React Native versions - New Architecture (Fabric) enabled
#### Supported React Native versions - New Architecture NOT enabled